packages/ember-views/lib/views/view.js

```diff
@@ -1305,6 +1305,10 @@ var View = CoreView.extend(
     }
 
     this._super.apply(this, arguments);
+
+    if (!this._viewRegistry) {
```

Comment:
@igorT - Not all view creation goes through the container via a fully booted application (most specifically those created in tests).  It is possible that we might not care if those end up in `Ember.View.views` though...